src/components/LayerInfosDialog/LayerInfosDialog.js

```diff
@@ -49,6 +49,7 @@ function LayerInfosDialog(props) {
       typeof component === 'string' ? layerInfos[component] : component;
     body = (
       <LayerInfoComponent
+        t={t}
```

Comment:
Is it possible to use https://react.i18next.com/latest/usetranslation-hook in our layer info components to avoid passing the t-function arround?

Reply:
The problem I encountered is that I am preparing further topics in parent repo, that will encapsulate this repo as webcomponent. There I cannot access this i18 instance to use the useTranslation hook.